test/Elastic.Apm.AspNetCore.Tests/MicrosoftExtensionsConfigTests.cs

```diff
@@ -152,7 +157,7 @@ public MicrosoftExtensionsConfigIntegrationTests(WebApplicationFactory<Startup>
 			var capturedPayload = new MockPayloadSender();
 
 			var config = new MicrosoftExtensionsConfig(
-				MicrosoftExtensionsConfigTests.GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_invalid.json"), _logger);
+				MicrosoftExtensionsConfigTests.GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_invalid.json"), _logger, "test");
```

Comment:
Please see my comment above regarding adding an `internal` constructor overload to `MicrosoftExtensionsConfig`.